mlrun/api/crud/model_endpoints.py

```diff
@@ -456,6 +481,12 @@ def build_kv_cursor_filter_expression(
     return " AND ".join(filter_expression)
 
 
+def parse_store_prefix(store_prefix: str):
+    scheme, path = store_prefix.split("///", 1)
```

Comment:
should use the same func as we use in feature store (w url parse and extract endpoint as well)